### PR TITLE
Add material theme

### PR DESCRIPTION
**Pull request by paulathevalley**
Add material theme. 
Default `victory-label` styles to match the grayscale theme.
Remove existing grayscale theme since it is the default for all the components, see https://github.com/FormidableLabs/victory-chart/pull/286

/cc @ebrillhart @boygirl @kylecesmat

### CODE REVIEW

File: src/victory-theme/material.js [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+const colors = [
+  deepOrange600,
+  yellow200,
+  lime300,
+  lightGreen500,
+  teal700,
+  cyan900
+];
+const blueGrey50 = "#ECEFF1";
+// const blueGrey100 = "#CFD8DC";
```
Comment (kylecesmat) [inline src/victory-theme/material.js]: are these comments purposeful?

File: src/victory-util/style.js [hunk 2]
```diff
@@ -47,7 +47,7 @@ export default {
   getColorScale(name) {
     const scales = {
       greyscale: [
-        "#7d7d7d", "#5e5e5e", "#969696", "#bdbdbd", "#000000"
+        "#f7f7f7", "#cccccc", "#969696", "#636363", "#252525"
```
Comment (kylecesmat) [inline src/victory-util/style.js:50]: nice! replacing #000 is a good move, thats too dark!

File: src/victory-theme/material.js (hunk 1 above)
Comment (paulathevalley) [inline src/victory-theme/material.js]: Whoops! Good catch, I will remove these.

**Comment by kylecesmat** [comment]
Looks great!

**Comment by ebrillhart** [comment]
looks good to me pending approval from @boygirl!

**Comment by boygirl** [comment]
@paulathevalley Approved!

**Pull request merged**